### DISCUSSION TITLE
compilation encountered a very confusing error..

### DISCUSSION THREAD

**Discussion by enp6** [rust9x/rust#26] [Q&A] [answered]
The prompt "=note: LINK: fatal error LNK1181: cannot open input file 'windows. 0.48.0. lib'" during compilation made me very confused...

I used these commands to compile the project

rustup default rust9x
cargo build --release --target=i686-rust9x-windows-msvc

**Comment by seritools** [comment, marked as answer]
https://github.com/rust9x/rust/wiki/Limitations#windows-rs-windows-sys upgrade to windows-rs 0.53.0 or higher, or check out the forked branches linked there, to apply a similar fix to 0.48.0 👍
